addons/l10n_be/data/account_fiscal_position_tax_template_data.xml

```diff
@@ -152,71 +152,66 @@
             <field name="tax_dest_id" ref="attn_VAT-OUT-00-ROW"/>
         </record>
         <record id="afpttn_extracom_7" model="account.fiscal.position.tax.template">
-            <field name="position_id" ref="fiscal_position_template_2"/>
-            <field name="tax_src_id" ref="attn_VAT-OUT-21-S"/>
-            <field name="tax_dest_id" ref="attn_VAT-OUT-00-ROW"/>
-        </record>
-        <record id="afpttn_extracom_8" model="account.fiscal.position.tax.template">
             <field name="position_id" ref="fiscal_position_template_2"/>
             <field name="tax_src_id" ref="attn_VAT-OUT-21-L"/>
             <field name="tax_dest_id" ref="attn_VAT-OUT-00-ROW"/>
         </record>
-        <record id="afpttn_extracom_9" model="account.fiscal.position.tax.template">
```

Comment:
renumbering might be a bit of an overkill and not helpful when we would have to check the origin of the tax through xml_id

Reply:
OK, I thought it would be better to avoid having a unexplained gap in the sequence, but I see your point :slightly_smiling_face: